revscoring/utilities/tests/test_union_merge_observations.py

```diff
@@ -0,0 +1,48 @@
+import json
+import os.path
+import tempfile
+
+import revscoring.utilities.union_merge_observations
+
+
+def test_union():
+    # Get test fixtures.
+    data_dir = os.path.dirname(__file__) + "/data"
+    in_files = [
+        data_dir + "/labeled_revisions.json",
+        data_dir + "/labeled_foo.json",
+    ]
+
+    # Receive the output in a temporary file.
+    (fh, out_file) = tempfile.mkstemp()
```

Comment:
An io.StringIO?  I wanted to test the stuff in main() as well as union_merge, so not sure how I would do this.  I need to pass main() a string that gets passed to open() as a pathname.